client/src/main/java/org/eclipse/hono/client/impl/CommandResponseSenderImpl.java

```diff
@@ -113,19 +113,12 @@ static final String getTargetAddress(final String tenantId, final String replyId
     public Future<ProtonDelivery> sendCommandResponse(final CommandResponse commandResponse,
             final SpanContext context) {
 
+        Objects.requireNonNull(targetAddress);
```

Comment:
*targetAddress* is not a parameter and should therefore not be checked using `Objects.requireNull()`. In fact, the *create* method which is used to create instances of this class, makes sure that targetAddress will not be null already.

Reply:
@sophokles73 Done.